docs/pages/connect-your-client/tsh.mdx

```diff
@@ -917,24 +921,43 @@ c0a02222-70dc-4be8-9308-7b7901d2d600 kube alice                  Nov 26 20:36:16
 d0a04442-70dc-4be8-9308-7b7901d2d600 ssh  navin        test      Nov 26 16:36:16 UTC
 ```
 
-To play sessions, use the `tsh play <sessionid>` command. The screen will reset playing the interactive session with the timestamp in the upper right.
+### Playing recordings
+
+To play a session recording, run the `tsh play` command with the ID of a session
+as returned by `tsh recordings ls`: 
 
 ```code
 $ tsh play c0a02222-70dc-4be8-9308-7b7901d2d600
-root@ubuntu:/# df                                                                     2022-11-26T21:20:52.715Z
-Filesystem     1K-blocks    Used Available Use% Mounted on
-overlay         20959212 4387284  16571928  21% /
-tmpfs              65536       0     65536   0% /dev
-tmpfs            1982720       0   1982720   0% /sys/fs/cgroup
-/dev/nvme0n1p1  20959212 4387284  16571928  21% /etc/hosts
-shm                65536       0     65536   0% /dev/shm
-tmpfs            3410436      12   3410424   1% /run/secrets/kubernetes.io/serviceaccount
-tmpfs            1982720       0   1982720   0% /proc/acpi
-tmpfs            1982720       0   1982720   0% /sys/firmware
-root@ubuntu:/# exit
-exit
 ```
 
+Users of self-hosted Teleport clusters can also run `tsh play` with the path to
+a TAR file that contains a session recording (this requires access to the
+session recording backend):
+
+```code
+$ tsh play ./my-recording.tar
+```
+
+The `tsh play` command can print recordings in three formats, depending on the
+kind of resource the recorded session interacts with. To choose a format, use
+the `--format` flag of `tsh play`:
+
+| `--format` value | Supported resources | Description |
+|------------------|---------------------|-------------|
+| `pty` (default)  | Servers, Kubernetes clusters | `tsh` opens a pseudo-terminal to play each command executed in the session. |
+| `json` | Servers, Kubernetes clusters, applications, databases | `tsh` prints a JSON-serialized list of audit events, separated by newlines. |
+| `yaml` | Servers, Kubernetes clusters, applications, databases | `tsh` prints a YAML-serialized list of audit events, separated by `---` characters. |
+
+To view recordings of Windows desktop sessions, you must run the following
```

Comment:
I would just remove this. It doesn't work with v15.

Reply:
👍 Does this mean that `tsh recordings export` doesn't work for v15? Is there any more context we should add to account for the fact that this command is available in the `tsh` help text?

Reply:
That's correct, the command does not work for v15.

We're not taking any action right now other than waiting to see if anyone complains. We suspect this is a rarely-used feature and won't invest engineering time in fixing it until we see evidence that it's actually being used.